.github/workflows/test-download-artifacts.yml

```diff
@@ -10,9 +10,9 @@ on:
       - completed
 
 jobs:
-  process-artifacts:
+  test-download-artifacts:
     runs-on: ubuntu-latest
-    name: Process artifacts
+    name: test download-artifacts
```

Comment:
What's the relationship between this and the `download-artifacts` action? Is this going to replace that?

Reply:
that's my thinking, since this is more powerful. 🤷

Reply:
Do you think you could do the work of removing the script and moving the existing job over to it?

Reply:
Only once this is in :D

Reply:
OK, thanks :)